### ISSUE TITLE
Typescript definition error

### ISSUE THREAD

**Issue by hasanmumin** [zxpsuper/qrcode-with-logos#9]
I think your typescript definition is not correct, here is my usage and error message. 
```
import QrCodeWithLogo from 'qrcode-with-logos';


const qrcode = new QrCodeWithLogo({
content: "https://github.com/zxpsuper",
image: document.getElementById("logo"),
width: 470,
  logo: {
     src: "logo_round.png",
 },
});
```
Error message: 

> (alias) const QrCodeWithLogo: IQrCodeWithLogo
> import QrCodeWithLogo
> This expression is not constructable.
> Type 'IQrCodeWithLogo' has no construct signatures.

**Comment by nweat** [comment]
@hasanmumin  I am having the same issue. For now, I have to explicitly declare a type like so: 
```
const _QrCodeWithLogo: any = QrCodeWithLogo;
new _QrCodeWithLogo({...})
```

**Comment by hasanmumin** [comment]
thank you @nweat I know, I did the same thing :)

**Comment by zxpsuper** [comment]
> I think your typescript definition is not correct, here is my usage and error message.
> 
> ```
> import QrCodeWithLogo from 'qrcode-with-logos';
> 
> 
> const qrcode = new QrCodeWithLogo({
> content: "https://github.com/zxpsuper",
> image: document.getElementById("logo"),
> width: 470,
>   logo: {
>      src: "logo_round.png",
>  },
> });
> ```
> 
> Error message:
> 
> > (alias) const QrCodeWithLogo: IQrCodeWithLogo
> > import QrCodeWithLogo
> > This expression is not constructable.
> > Type 'IQrCodeWithLogo' has no construct signatures.

It has been solved in version@1.0.3.

**Comment by zxpsuper** [comment]
> @hasanmumin I am having the same issue. For now, I have to explicitly declare a type like so:
> 
> ```
> const _QrCodeWithLogo: any = QrCodeWithLogo;
> new _QrCodeWithLogo({...})
> ```

It has been solved in version@1.0.3.

**Comment by zxpsuper** [comment]
It has been solved in version@1.0.3.
